kaiko/src/adapter.ts

```diff
@@ -29,11 +29,13 @@ export const execute: ExecuteWithConfig<Config> = async (request, config) => {
 
   let inverse = false
   let url = `/spot_exchange_rate/${base}/${quote}`
-  if (quote === 'eth') {
+  if (includes.length === 0 && quote === 'eth') {
     url = `/spot_direct_exchange_rate/${base}/${quote}`
   } else if (includes.length > 0 && base === 'digg' && includes[0].toLowerCase() === 'wbtc') {
     inverse = true
     url = `/spot_direct_exchange_rate/wbtc/digg`
+  } else if (includes.length > 0 && includes[0].toLowerCase() === 'weth') {
+    url = `/spot_direct_exchange_rate/${base}/weth`
   }
```

Comment:
Just in case `quote` is "eth" and `includes` is not empty but does not have "weth".

```suggestion
  if (includes.length > 0 && base === 'digg' && includes[0].toLowerCase() === 'wbtc') {
    inverse = true
    url = `/spot_direct_exchange_rate/wbtc/digg`
  } else if (includes.length > 0 && includes[0].toLowerCase() === 'weth') {
    url = `/spot_direct_exchange_rate/${base}/weth`
  } else if (quote === 'eth') {
    url = `/spot_direct_exchange_rate/${base}/${quote}`
  }
```